config.md

```diff
@@ -373,6 +373,7 @@ For POSIX platforms, the configuration structure supports `hooks` for configurin
         Entries in the array contain the following properties:
         * **`path`** (string, REQUIRED) with similar semantics to [IEEE Std 1003.1-2008 `execv`'s *path*][ieee-1003.1-2008-functions-exec].
             This specification extends the IEEE standard in that **`path`** MUST be absolute.
+            Runtimes MUST resolve this value in the [runtime mount namespace](glossary.md#runtime-namespace).
```

Comment:
remove `mount` here to make it consistent with the link and the other usage in your changeset.

Reply:
ok, updated.